File: Code/Fluent.Java/src/adstack/components/GNSSReceiver.java
package adstack.components;

import adstack.models.IComponent;

public class GNSSReceiver implements IComponent {

	@Override
	public void start() {
        System.out.println("[GNSSReceiver] Starting.");
	}

	@Override
	public void stop() {
        System.out.println("[GNSSReceiver] Stopping.");
	}

}

File: Code/dotNetBasics/ConsoleApp/ADStack/Components/GNSSReceiver.cs
using ConsoleApp.ADStack.Models;

/// <summary>
/// GNSS/GPS receiver
/// </summary>
internal class GNSSReceiver : IComponent
{
    public void Start()
    {
        Console.WriteLine("[GNSSReceiver] Starting.");
    }

    public void Stop()
    {
        Console.WriteLine("[GNSSReceiver] Stopping.");
    }
}